replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicationSiteImpl.java

```diff
@@ -13,24 +13,17 @@
  */
 package org.codice.ditto.replication.api.impl.data;
 
-import java.util.Map;
 import org.codice.ditto.replication.api.data.ReplicationSite;
+import org.springframework.data.solr.core.mapping.Indexed;
+import org.springframework.data.solr.core.mapping.SolrDocument;
 
 /**
  * ReplicationSiteImpl represents a replication site and has methods that allow it to easily be
  * converted to, or from, a map.
  */
+@SolrDocument(solrCoreName = "replication_site")
```

Comment:
`solrCoreName` is deprecated. Use `collection`. 
https://docs.spring.io/spring-data/solr/docs/current/api/org/springframework/data/solr/core/mapping/SolrDocument.html